app/tests/test_questions.py

```diff
@@ -20,10 +21,11 @@ def test_get_question_failure(self):
         question_id = "99"
         seed = 123
 
-        question, status = questions.get_question(question_type, question_id, seed)
-
-        self.assertEqual("Question not found.", question.description)
-        self.assertEqual(404, status)
+        try:
+            question, status = questions.get_question(question_type, question_id, seed)
+            self.assertFalse(True)
```

Comment:
what on earth

Reply:
test should fail if exception isn't raised

Reply:
```suggestion
            self.assertFalse(True)  # We expect the above line to throw an exception, so this code should only be hit if it fails to throw
```

Reply:
Or something